README.md

```diff
@@ -145,6 +145,8 @@ Vigil will be reachable from `http://localhost:8080`.
 
 Use the sample [config.cfg](https://github.com/valeriansaliou/vigil/blob/master/config.cfg) configuration file and adjust it to your own environment.
 
+Note: you can use environment variables with string interpolation in your configuration file. For example: `manager_token: ${VIGIL_MANAGER_TOKEN}`.
```

Comment:
is the example syntax wrong ?
Should it not be `=` ?

Reply:
I just tested interpolation and the documented syntax definitely works as expected.

Reply:
Yes but the toml syntax is not
```toml
manager_token: ${VIGIL_MANAGER_TOKEN}
```
But 
```toml
manager token = ${VIGIL_MANAGER_TOKEN}
```
Right

Reply:
Ah, good catch! Fixed in 78100cae99a461a11a65444e918403533491f698

Reply:
Thank you!
One good example to show would be
`"icmp://${MY_SECRET_ADDR}"`